hw/ip/sram_ctrl/doc/dv/index.md

```diff
@@ -39,32 +41,79 @@ The following utilities provide generic helper tasks and functions to perform ac
 * [csr_utils_pkg]({{< relref "hw/dv/sv/csr_utils/README.md" >}})
 
 ### Compile-time configurations
-[list compile time configurations, if any and what are they used for]
+Two compile-time configurations are tested:
+* `sram_ctrl_main` - this configuration uses a 15-bit wide address space to
+  emulate the main SRAM used in the full design
+* `sram_ctrl_ret` - this configuration uses a 10-bit-wide address space to
+  emulate the retention SRAM used in the full design
+
+A macro-define `SRAM_ADDR_WIDTH` is defined as a build option in
+[`sram_ctrl_base_sim_cfg.hjson`]({{< relref "hw/ip/sram_ctrl/dv/sram_ctrl_base_sim_cfg.hjson" >}}),
+which is used to set the correct compile-time settings for each configuration.
 
 ### Global types & methods
 All common types and methods defined at the package level can be found in
 `sram_ctrl_env_pkg`. Some of them in use are:
 ```systemverilog
-[list a few parameters, types & methods; no need to mention all]
+// number of bits in a full KDI transactions
+parameter int KDI_DATA_SIZE = 1 + otp_ctrl_pkg::SramKeyWidth + otp_ctrl_pkg::SramNonceWidth;
+
+// a completed KDI transaction takes 4 cycles to be fully synchronized into the
+// SRAM clock domain
+parameter int KDI_PROPAGATION_CYCLES = 4;
+
+// a LC escalation request needs 3 cycles to be fully propagated through the DUT
+parameter int LC_ESCALATION_PROPAGATION_CYCLES = 3;
+
+typedef virtual mem_bkdr_if #(.MEM_PARITY(1)) mem_bkdr_vif;
+
+typedef enum bit {
+  SramCtrlRenewScrKey = 0,
+  SramCtrlInit        = 1
+} sram_ctrl_e;
+
+typedef enum bit [1:0] {
+  SramCtrlError           = 0,
+  SramCtrlEscalated       = 1,
+  SramCtrlScrKeyValid     = 2,
+  SramCtrlScrKeySeedValid = 3
+} sram_ctrl_status_e;
 ```
 ### TL_agent
 SRAM_CTRL testbench instantiates (already handled in CIP base env) [tl_agent]({{< relref "hw/dv/sv/tl_agent/README.md" >}})
 which provides the ability to drive and independently monitor random traffic via
 TL host interface into SRAM_CTRL device.
 
-### UVC/agent 1
-[Describe here or add link to its README]
+### Alert agents
+The SRAM_CTRL testbench instantiates 2 [alert agents]({{< relref "hw/dv/sv/alert_esc_agent/README.md" >}}) for:
+* fatal_intg_error - signals a transmission integrity error
+* fatal_parity_error - signals an internal parity error in the memory primitive
+
+The alert agents provide the ability to independently drive and monitor both
+alert handshakes.
+
+### SRAM_CTRL_LC_IF
+The SRAM_CTRL IP has a simple sideband interface to the LC_CTRL to receive escalation requests.
+This interface initializes the escalation bus and utilizes a simple task to
```

Comment:
I thought the recommendation is to have sentences in a single line.

Reply:
thanks for catching that - this was most likely due to my editor auto-formatting line length...will fix!